.github/patches/extensions/sqlite_scanner/add_gettable_sample.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/src/include/storage/sqlite_table_entry.hpp b/src/include/storage/sqlite_table_entry.hpp
+index b08319b..b13d120 100644
+--- a/src/include/storage/sqlite_table_entry.hpp
++++ b/src/include/storage/sqlite_table_entry.hpp
+@@ -21,6 +21,8 @@ public:
+ public:
+ 	unique_ptr<BaseStatistics> GetStatistics(ClientContext &context, column_t column_id) override;
+ 
++	unique_ptr<BlockingSample> GetSample() override;
++
+ 	TableFunction GetScanFunction(ClientContext &context, unique_ptr<FunctionData> &bind_data) override;
+ 
+ 	TableStorageInfo GetStorageInfo(ClientContext &context) override;
+diff --git a/src/storage/sqlite_table_entry.cpp b/src/storage/sqlite_table_entry.cpp
+index 47378b0..e0bf354 100644
+--- a/src/storage/sqlite_table_entry.cpp
++++ b/src/storage/sqlite_table_entry.cpp
+@@ -16,6 +16,10 @@ unique_ptr<BaseStatistics> SQLiteTableEntry::GetStatistics(ClientContext &contex
+ 	return nullptr;
+ }
+ 
++unique_ptr<BlockingSample> SQLiteTableEntry::GetSample() {
++	return nullptr;
```

Comment:
Can we make `GetSample()` return a `nullptr` by default? Then we don't need to patch the extensions